smac/facade/roar_facade.py

```diff
@@ -70,6 +73,9 @@ def __init__(self,
             Random number generator
         run_id: int, (default: 1)
             Run ID will be used as subfolder for output_dir.
+        random_configuration_chooser
```

Comment:
What does the random_configuration_chooser do for ROAR? It should not be needed at all?